enforcer-parent/enforcer/src/main/resources/Dockerfile.ubuntu

```diff
@@ -56,6 +56,7 @@ RUN \
     groupadd --system -g ${MG_USER_GROUP_ID} ${MG_USER_GROUP} \
     && useradd --system --create-home --home-dir ${MG_USER_HOME} --no-log-init -g ${MG_USER_GROUP_ID} -u ${MG_USER_ID} ${MG_USER} \
     && mkdir ${MG_USER_HOME}/logs && chown -R ${MG_USER}:${MG_USER_GROUP} ${MG_USER_HOME}/logs  \
+    && mkdir -p ${MG_USER_HOME}/lib/dropins \
```

Comment:
Same here.. Let's verify the functionalities too.

Reply:
fixed